src/common/options.cc

```diff
@@ -6958,6 +6958,10 @@ std::vector<Option> get_mds_options() {
     .set_default(15)
     .set_description("tolerance in seconds for missed MDS beacons to monitors"),
 
+    Option("mds_heartbeat_grace", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    .set_default(15)
```

Comment:
Is this too small

Reply:
It's the current default and I think it's appropriate.